2023-02-02 05:44:26,872 INFO    MainThread:10380 [wandb_setup.py:_flush():75] Loading settings from C:\Users\bhata\.config\wandb\settings
2023-02-02 05:44:26,872 INFO    MainThread:10380 [wandb_setup.py:_flush():75] Loading settings from C:\aiornot\notebooks\wandb\settings
2023-02-02 05:44:26,872 INFO    MainThread:10380 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2023-02-02 05:44:26,872 INFO    MainThread:10380 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-02 05:44:26,872 INFO    MainThread:10380 [wandb_init.py:_log_setup():386] Logging user logs to C:\aiornot\notebooks\wandb\run-20230202_054426-32qeond4\logs\debug.log
2023-02-02 05:44:26,873 INFO    MainThread:10380 [wandb_init.py:_log_setup():387] Logging internal logs to C:\aiornot\notebooks\wandb\run-20230202_054426-32qeond4\logs\debug-internal.log
2023-02-02 05:44:26,873 INFO    MainThread:10380 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001957EE1DCA0>
2023-02-02 05:44:26,873 INFO    MainThread:10380 [wandb_init.py:init():420] calling init triggers
2023-02-02 05:44:26,874 INFO    MainThread:10380 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2023-02-02 05:44:26,874 INFO    MainThread:10380 [wandb_init.py:init():471] starting backend
2023-02-02 05:44:26,874 INFO    MainThread:10380 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2023-02-02 05:44:26,877 INFO    MainThread:10380 [backend.py:ensure_launched():219] starting backend process...
2023-02-02 05:44:26,945 INFO    MainThread:10380 [backend.py:ensure_launched():224] started backend process with pid: 16620
2023-02-02 05:44:26,946 INFO    MainThread:10380 [wandb_init.py:init():480] backend started and connected
2023-02-02 05:44:26,958 INFO    MainThread:10380 [wandb_run.py:_label_probe_notebook():921] probe notebook
2023-02-02 05:44:26,960 INFO    MainThread:10380 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-02 05:44:26,960 INFO    MainThread:10380 [wandb_init.py:init():550] updated telemetry
2023-02-02 05:44:27,004 INFO    MainThread:10380 [wandb_init.py:init():581] communicating current version
2023-02-02 05:44:28,333 INFO    MainThread:10380 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.13.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-02-02 05:44:28,334 INFO    MainThread:10380 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2023-02-02 05:44:28,804 INFO    MainThread:10380 [wandb_init.py:init():624] starting run threads in backend
2023-02-02 05:44:33,825 INFO    MainThread:10380 [wandb_run.py:_console_start():1827] atexit reg
2023-02-02 05:44:33,826 INFO    MainThread:10380 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2023-02-02 05:44:33,826 INFO    MainThread:10380 [wandb_run.py:_redirect():1738] Wrapping output streams.
2023-02-02 05:44:33,829 INFO    MainThread:10380 [wandb_run.py:_redirect():1762] Redirects installed.
2023-02-02 05:44:33,829 INFO    MainThread:10380 [wandb_init.py:init():651] run started, returning control to user process
2023-02-02 05:44:46,545 INFO    MainThread:10380 [wandb_init.py:_pause_backend():299] pausing backend
2023-02-02 05:44:46,546 INFO    MainThread:10380 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-02-02 05:45:18,695 INFO    MainThread:10380 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2023-02-02 05:45:18,696 INFO    MainThread:10380 [wandb_run.py:_restore():1769] restore
2023-02-02 05:45:18,857 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 876
}

2023-02-02 05:45:19,212 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 876
}

2023-02-02 05:45:20,117 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10487
}

2023-02-02 05:45:20,226 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10514
}

2023-02-02 05:45:20,335 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10514
}

2023-02-02 05:45:20,450 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10514
}

2023-02-02 05:45:20,555 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:20,667 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:20,777 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:20,885 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:20,993 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:21,103 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

2023-02-02 05:45:21,212 INFO    MainThread:10380 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10514
  total_bytes: 10514
}

